simplefs/simplefs.go

```diff
@@ -989,30 +971,12 @@ func (k *SimpleFS) SimpleFSSymlink(ctx context.Context, arg keybase1.SimpleFSSym
 	}
 	defer func() { k.doneSyncOp(ctx, err) }()
 
-	// Get root FS of dst
-	t, tlfName, restOfDstPath, finalDstElem, err := remoteTlfAndPath(arg.Dest)
-	if err != nil {
-		return err
-	}
-	tlfHandle, err := libkbfs.GetHandleFromFolderNameAndType(
-		ctx, k.config.KBPKI(), k.config.MDOps(), tlfName, t)
-	if err != nil {
-		return err
-	}
-	fs, err := libfs.NewFS(
-		ctx, k.config, tlfHandle, libkbfs.MasterBranch, "", "",
-		keybase1.MDPriorityNormal)
-	if err != nil {
-		return err
-	}
-	srcRawPath, err := rawPathFromPath(arg.Src)
+	dstFS, finalDstElem, err := k.getFS(ctx, arg.Dest)
```

Comment:
Please check here too that the path type of Dest is only KBFS.  (I know you check on in the command implementation, but there could be other callers of this RPC, like the GUI eventually.)